src/com/android/launcher3/Utilities.java

```diff
@@ -882,6 +879,8 @@ public static void openURLinBrowser(Context context, String url, Rect sourceBoun
         } catch (ActivityNotFoundException exc) {
             // Believe me, this actually happens.
             Toast.makeText(context, R.string.error_no_browser, Toast.LENGTH_SHORT).show();
+        } catch (NullPointerException e) {
```

Comment:
Instead of catching an NPE here check the URL passed in at the beginning using TextUtils

Reply:
Done